projects/tad/index.md

```diff
@@ -0,0 +1,152 @@
+# Transparency of algorithmic decision making
+
+This document contains a checklist with requirements for tools we could use to help with the
+transparency of algorithmic decision making.
+
+The requirements are based on:
+
+- **ISO 25010 standard:** This standard defines the eight quality characteristics and provides a
+framework for evaluating software quality ([source](https://www.iso.org/standard/35733.html)).
+- **Industry best practices:** This includes a broad range of recommendations and guidelines for IT tool
+development and implementation.
+- **Common IT tool requirements:** This information was gathered by analyzing various sources, such as
+documentation from popular IT tools, user reviews, and articles from reputable tech publications
+that discuss essential features and functionalities expected from different types of IT tools.
+- **Internal discussion and common sense**: While above sources are already exhaustive, we also
+used team discussions and our own knowledge.
+
+## Overview of requirements
+
+The requirements have been given a priority based on the [MoSCoW](https://en.wikipedia.org/wiki/MoSCoW_method)
+scale to allow for tool comparison.
+
+### Functionality
+
+| Requirement                                                                                                                                            | Priority |
+|--------------------------------------------------------------------------------------------------------------------------------------------------------|----------|
+| The tool allows users to perform technical tests on an algorithm or model.                                                                             | M        |
```

Comment:
Technical tests can mean a lot of things. Maybe we can be more specific, for example technical tests to asses model performance, model bias, model fairness. Also, many of the technical tests must include a dataset, so a relevant dataset can also be input for technical tests.